### PR TITLE
chore: bump Augury to version 1.0.3, and some 1.0.2 changelog tweaks

### PR DESCRIPTION
**Pull request by ericjim**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # Change Log
 
-## 1.0.2
+## [1.0.3](https://github.com/rangle/augury/tree/1.0.3)
```
Comment (ericjim) [inline CHANGELOG.md]: This link currently doesn't exist @sumitarora

Comment (sumitarora) [inline CHANGELOG.md]: yes before release we will create tag then it will be there

**Comment by sumitarora** [comment]
rebase with dev :shipit: 🚀

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+* Update Augury to Angular 2.0.0-rc.3 ([450](https://github.com/rangle/augury/issues/450))
```
Comment (igor-ka) [inline CHANGELOG.md]: I think it should be updating Augury to support RC 3 applications or something like that instead

Comment (ericjim) [inline CHANGELOG.md]: Just for my understanding, Augury is built with RC3, and it also supports applications that are written in RC3. @sumitarora

Comment (sumitarora) [inline CHANGELOG.md]: yes it does

**Pull request merged**